ta/pkcs11/src/pkcs11_token.c

```diff
@@ -27,6 +27,8 @@
 /* Static allocation of tokens runtime instances (reset to 0 at load) */
 struct ck_token ck_token[TOKEN_COUNT];
 
+static struct client_list pkcs11_client_list;
```

Comment:
Any reason why `TAILQ_INIT()` isn't used here?

Reply:
Sorry, I meant `TAILQ_HEAD_INITIALIZER()`

Reply:
oh, yes. Better `TAILQ_HEAD_INITIALIZER()` here than `TAILQ_INIT()` in `pkcs11_inti()` function.
I'll fix. Thanks.

Reply:
Missed this one?

Reply:
indeed, sorry.